main.py

```diff
@@ -1,10 +1,10 @@
 import math
-import tkFont
+from tkinter import font as tkFont
 import numpy as np
 import cv2  # required 3+
-import Tkinter as tk
-import thread
-import Queue
+import tkinter as tk
+import _thread as thread
```

Comment:
I think using https://docs.python.org/3/library/threading.html#threading.Thread would be a better idea? 😉

Reply:
I was thinking of changing the code as minimum as possible. I implemented using the threading.thread. please check.